Compiler breaks on Renderer type definition

Compiling our project that uses ngx-markdown (6.2.0) breaks on the type definition of marked.Renderer.list. Log excerpt:

```
ERROR in node_modules/ngx-markdown/src/marked-options.d.ts(41,5): error TS2416: Property 'renderer' in type 'MarkedOptions' is not assignable to the same property in base type 'MarkedOptions'.
  Type 'marked.Renderer' is not assignable to type 'marked.Renderer'. Two different types with this name exist, but they are unrelated.
    Types of property 'list' are incompatible.
      Type '(body: string, ordered: boolean, start: number) => string' is not assignable to type '(body: string, ordered: boolean) => string'.

```

This appears to be an issue in the underlying @types/marked that has been fixed in the source (see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/29279). Could you update ngx-markdown with the fixed definition?

--- a/lib/package.json
+++ b/lib/package.json
@@ -23,8 +23,8 @@
     "prism.js"
   ],
   "dependencies": {
-    "@types/marked": "^0.4.0",
-    "marked": "^0.5.0",
+    "@types/marked": "^0.4.2",
+    "marked": "^0.5.1",
     "prismjs": "^1.15.0"
   },
   "peerDependencies": {

